### ISSUE TITLE
Extra indent added after multi-line destructured assignment

### ISSUE THREAD

**Issue by mjbvz** [microsoft/TypeScript#19167]
From https://github.com/Microsoft/vscode/issues/36248

**TypeScript Version:**  2.6.0-dev.20171013

**Code**
Format the code:

```ts
const {
    a
} = parseInt(
    '12'
)
```

**Expected behavior:**

```js
const {
    a
} = parseInt(
    '12'
)
```

**Actual behavior:**
Extra indent added:

```js
const {
    a
} = parseInt(
        '12'
    )
```

Similar to #17699

**Comment by aleclarson** [comment]
This deserves higher priority now that [typescript-tslint-plugin](https://github.com/Microsoft/typescript-tslint-plugin) is affected. https://github.com/Microsoft/TypeScript/issues/31237

**Comment by yume-chan** [comment]
I can verify that this one has been fixed by #28340.

@aleclarson your issue is slightly different so the formatter need to be adjusted case by case.

**Comment by mjbvz** [comment]
Confirmed as fixed in TS 3.7.5 (and likely earlier versions too as @yume-chan noted)
